compiler/runtime/Makefile

```diff
@@ -1,12 +1,13 @@
 MEMSIZE32=1073741824
 
-COMMONFLAGS=-Wall -Wextra -Wno-sign-conversion -Wno-sometimes-uninitialized -Wno-c2x-extensions -Wsign-compare -Wextra-semi-stmt
+COMMONFLAGS=-Werror -Wall -Wextra -Wno-sign-conversion -Wno-sometimes-uninitialized -Wno-c2x-extensions -Wsign-compare -Wextra-semi-stmt
 # To add: -Wcast-align -Wcast-qual -Wimplicit-int-conversion -Wmissing-noreturn -Wpadded -Wreserved-identifier -Wshorten-64-to-32 -Wtautological-unsigned-zero-compare
 # -Watomic-implicit-seq-cst ?
 # If using -Weverything, you may want to add -Wno-declaration-after-statement -Wno-missing-prototypes -Wno-missing-variable-declarations -Wno-shadow -Wno-strict-prototypes -Wno-zero-length-array -Wno-unreachable-code-break -Wno-unreachable-code-return
 OLEVEL=-O2
 CC32FLAGS=-DSKIP32 --target=wasm32 -emit-llvm -nostdlibinc $(COMMONFLAGS)
 CC64FLAGS=$(OLEVEL) -DSKIP64 $(COMMONFLAGS)
+CXX64FLAGS=$(OLEVEL) $(COMMONFLAGS)
```

Comment:
I have no idea either.

I agree, there was actually no need for a C++-specific variable, especially because we do want `-DSKIP64` as well but I enforced it in `runtime64_specific.cpp`.

Btw this commit was part of #148 on which I rebased this PR. I won't change anything there.

Reply:
I changed to share the flags + ensure `runtime64_specific.cpp` is given `-DSKIP64`